packages/contentful--create-contentful-app/src/getTemplateSource.ts

```diff
@@ -0,0 +1,108 @@
+
+import fetch from 'node-fetch';
+import inquirer from 'inquirer';
+import chalk from 'chalk';
+import { track } from './analytics';
+import { CLIOptions, ContentfulExample } from './types';
+import { highlight, warn, wrapInBlanks } from './logger';
+
+const EXAMPLES_PATH = 'contentful/apps/examples/';
+
+function isContentfulTemplate(url: string) {
+  return Object.values(ContentfulExample).some((t) => url.includes(EXAMPLES_PATH + t));
+}
+
+async function getGithubFolderNames(username: string, repo: string, path: string) {
+  const url = `https://api.github.com/repos/${username}/${repo}/contents/${path}`;
```

Comment:
Maybe just hardcode the url here? I think it won't change in the nearest observable future.

Reply:
Same as above, was just copy-paste but I can change it